src/index.js

```diff
@@ -0,0 +1,71 @@
+import './style.css';
+import getAllSeaFoodMeals from './modules/meals.js';
+import { addLikeOnEvent, populateCurrentLikes } from './modules/cardlikes.js';
+import logo from '../assets/logo-kan.png';
+import like from '../assets/like2.png';
+import getItem from './modules/reservation.js';
+
+const logoImg = document.querySelector('.logo');
+logoImg.src = logo;
+
+let allSeafoods = [];
+
+(async () => {
+  try {
+    allSeafoods = await getAllSeaFoodMeals();
+  } catch (error) {
+    /* eslint-disable */
+    console.error(error.message);
+    /* eslint-enable */
+  }
+})();
+
+const setEventForReservation = (btnReserve, mealId) => {
+  btnReserve.addEventListener('click', (e) => {
+    if (e.target.classList.contains('reserve')) {
+      getItem(mealId);
+    }
+  });
+};
+
+const showCards = (foodArray) => {
+  const container = document.querySelector('.container');
+  container.innerHTML = '';
+
+  for (let index = 0; index < foodArray.length; index += 1) {
+    const cardDiv = document.createElement('div');
+    cardDiv.classList.add('flex', 'card');
+    cardDiv.innerHTML = `<h1>${foodArray[index].strMeal}</h1>
+               <h6>${foodArray[index].idMeal}</h6>
+               <img src="${foodArray[index].strMealThumb}" alt="food image" >
+               <div class="likes flex">
+                <img class="like-img img${index}" src="" alt="like image">
+                <h4><span id = "s${foodArray[index].idMeal}">0</span> likes</h4>
+               </div>
+               <div class="flex crd-btns">
+                <button class="comment">Comments</button>
```

Comment:
Hello, again, So, in this case we were only 2 people (@ghermaico135 as Student C  and me @porag-m06 - as Student A) on the team. Then @maryam0007 as student B Joined. And we divided and assigned our tasks according to the instructions. 

Implementation of the Comment section was assigned on @maryam0007, however, unfortunately, due to her illness she could not complete any of the assigned tasks according to the instructions. 

Therefore, that part is unfinished. In this scenario is mandatory for us 2 (As students A and C ) to complete the part of Student B as well to get approved? It would be great if you could shed some light on it, please.   

As, technically we are only 2 people team.  Thanks

Reply:
![image](https://github.com/porag-m06/Kanban-Restaurant/assets/15729467/9089ed25-b574-4453-bd33-d14abbb63142)

Reply:
https://github.com/porag-m06/Kanban-Restaurant/projects/1

Reply:
- [ ] As suggested by the previous reviewer, please implement the comments section so that users can post comments. It's so unfortunate that Student B fell sick but when such a thing occurs, you need to contact the Student Success team and forge a way forward together. Deciding on a course of action individually is not professional and could lead to disastrous results on a job. Please reach out to The Student Success Team at your earliest convenience. 👍🏾